resources/development-guidelines/good-commit-practice.md

```diff
@@ -63,15 +63,15 @@ The cardinal rule for creating good commits is to ensure there is only one
 rule:
 
 *   The smaller the amount of code being changed, the quicker and easier it is
-    to review and identify potential flaws.
+  to review and identify potential flaws.
 *   If a change is found to be flawed later, it may be necessary to revert the
-    broken commit. This is much easier to do if there are not other unrelated
-    code changes entangled with the original commit.
+  broken commit. This is much easier to do if there are not other unrelated
+  code changes entangled with the original commit.
 *   When troubleshooting problems using Git's [bisect][bisect] capability,
-    small well defined changes will aid in isolating exactly where the code
-    problem was introduced.
+  small well defined changes will aid in isolating exactly where the code
+  problem was introduced.
 *   When browsing history using Git annotate/blame, small well defined changes
-    also aid in isolating exactly where & why a piece of code came from.
+  also aid in isolating exactly where & why a piece of code came from.
```

Comment:
This tables indentation is inconsistent with the rest, would you please re-review it? :)